Error: execute permission denied for /usr/bin/strace
Error: no suitable strace executable found
=== STRACE OUTPUT ===

=== LSOF OUTPUT (open files/network) ===
COMMAND   PID USER   FD   TYPE DEVICE SIZE/OFF    NODE NAME
python  33822 kali  cwd    DIR    8,1     4096 1838335 /home/kali/linux-malware-sandbox
python  33822 kali  rtd    DIR    8,1     4096       2 /
python  33822 kali  txt    REG    8,1  6815960  403120 /usr/bin/python3.11
python  33822 kali  mem    REG    8,1   194552 1594146 /usr/lib/x86_64-linux-gnu/liblzma.so.5.6.2
python  33822 kali  mem    REG    8,1  5706872 1593986 /usr/lib/x86_64-linux-gnu/libcrypto.so.3
python  33822 kali  mem    REG    8,1    44888  565570 /usr/lib/python3.11/lib-dynload/_json.cpython-311-x86_64-linux-gnu.so
python  33822 kali  mem    REG    8,1    74688 1592343 /usr/lib/x86_64-linux-gnu/libbz2.so.1.0.4
python  33822 kali  mem    REG    8,1   129096 1600676 /usr/lib/x86_64-linux-gnu/ossl-modules/legacy.so
python  33822 kali  mem    REG    8,1   813048 1594406 /usr/lib/x86_64-linux-gnu/libzstd.so.1.5.6
python  33822 kali  mem    REG    8,1    44920  565604 /usr/lib/python3.11/lib-dynload/_lzma.cpython-311-x86_64-linux-gnu.so
python  33822 kali  mem    REG    8,1    63536  565582 /usr/lib/python3.11/lib-dynload/_hashlib.cpython-311-x86_64-linux-gnu.so
python  33822 kali  mem    REG    8,1  3052896  562771 /usr/lib/locale/locale-archive
python  33822 kali  mem    REG    8,1    14616 1447345 /usr/lib/python3/dist-packages/markupsafe/_speedups.cpython-311-x86_64-linux-gnu.so
python  33822 kali  mem    REG    8,1  1933688 1592275 /usr/lib/x86_64-linux-gnu/libc.so.6
python  33822 kali  mem    REG    8,1   174176 1592399 /usr/lib/x86_64-linux-gnu/libexpat.so.1.9.2
python  33822 kali  mem    REG    8,1   121280 1593942 /usr/lib/x86_64-linux-gnu/libz.so.1.3.1
python  33822 kali  mem    REG    8,1   919768 1594350 /usr/lib/x86_64-linux-gnu/libm.so.6
python  33822 kali  mem    REG    8,1    14336  565606 /usr/lib/python3.11/lib-dynload/_typing.cpython-311-x86_64-linux-gnu.so
python  33822 kali  mem    REG    8,1    27832  565577 /usr/lib/python3.11/lib-dynload/_bz2.cpython-311-x86_64-linux-gnu.so
python  33822 kali  mem    REG    8,1    27028 1599712 /usr/lib/x86_64-linux-gnu/gconv/gconv-modules.cache
python  33822 kali  mem    REG    8,1   210728 1593685 /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
python  33822 kali    0u   CHR  136,0      0t0       3 /dev/pts/0
python  33822 kali    1u   CHR  136,0      0t0       3 /dev/pts/0
python  33822 kali    2u   CHR  136,0      0t0       3 /dev/pts/0
python  33822 kali    3r  FIFO   0,14      0t0   73993 pipe
python  33822 kali    5w   REG    8,1       96 1836664 /home/kali/linux-malware-sandbox/logs/log_20250420_232502.txt